src/components/app/components/Navbar.js

```diff
@@ -73,21 +67,30 @@ class Navbar extends React.Component {
                     theme={NAV_THEME}
                     selectedKeys={[location.pathname]}
                 >
-                    {
-                        items.map(item => {
-                            const { key } = item;
-                            const className = key === AVATAR ? CONTAINER : '';
-
-                            return (
-                                <Item className={className} key={key}>{item}</Item>
-                            );
-                        })
-                    }
+                    <Item key={AVATAR}>
+                        <AppLink key={AVATAR} to={HOME}>
+                            <Avatar className={AVATAR} src={SOURCE} size={SIZE} shape={SHAPE} />
+                        </AppLink>
+                    </Item>
+                    <Item key={MARKET}>
+                        <AppLink key={MARKET} to={MARKET}>{MARKET_TEXT}</AppLink>
+                    </Item>
```

Comment:
Why did you remove the `items.map` here?

Reply:
At a point my code ran into error I decided to split. I was going to put that back